components/chart/drilldown.md

```diff
@@ -0,0 +1,205 @@
+---
+title: DrillDown
+page_title: DrillDown 
+description: The DrillDown Chart feature allows the users to explore hierarchical data by initially displaying summarized information and to "drill down" into specific categories or data points for more detailed insights.
+slug: chart-drilldown
+tags: telerik,blazor,chart,drill,down,drilldown
+published: true
+position: 55
+---
+
+# DrillDown Charts
+
+The Telerik UI for Blazor Chart supports drilldown functionality for exploring data.
+
+The drill-down feature allows the users to click on a point (for example, bar, pie segment, and more) to navigate to a different view that contains finer-grained data about the selected data point, like breakdown by product of the selected category. The view hierarchy can be displayed in a [Breadcrumb]({%slug breadcrumb-overview%}) for easy navigation back to previous views.
+
+## Configuring DrillDown Charts
+
+To configure Chart series for drill-down:
+
+1. Prepare the data in the appropriate format. Each series data that will be drilled-down must contain a property of type `ChartSeriesDescriptor`. The descriptor includes all the parameters of the `ChartSeries` tag and acts as a container holding information about the series displayed upon user-initiated drill-down.
+1. Specify the drilldown field (the `ChartSeriesDescriptor` field) of the series data by using the `ChartSeries.DrilldownField` or `ChartSeriesDescriptor.DrilldownField` property.
+
+````CSHTML
+@* Configuring the DrillDown Chart *@
+
+<TelerikChart>
+    <ChartSeriesItems>
+        <ChartSeries Type="ChartSeriesType.Column"
+                     Name="Total Sales By Company"
+                     Data="@Data"
+                     Field="@nameof(CompanyModel.Sales)"
+                     CategoryField="@nameof(CompanyModel.Name)"
+                     DrilldownField="@nameof(CompanyModel.Details)">
+        </ChartSeries>
+    </ChartSeriesItems>
+</TelerikChart>
+
+@code {
+    public List<CompanyModel> Data { get; set; } = new List<CompanyModel>();
```

Comment:
Use private accessors.
The data can be hard-coded and defined outside OnInitialized. Or, switch the places of the two methods, so that the more important code is on top.